### PR TITLE
Favourite Workflow-detail design 

### PR DESCRIPTION
**Pull request by sagar15795**
I have implemented the same Detail layout functionality  as I show in the dashboard.

### CODE REVIEW

**Comment by sagar15795** [comment]
@therajanmaurya 

Please review it. 
Tell me if there is any suggestion or change .

Thanks

File: app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflow/FavouriteWorkflowsFragment.java [hunk 1]
```diff
@@ -80,12 +90,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
...
 
         mRecyclerView.setLayoutManager(layoutManager);
         mRecyclerView.setItemAnimator(new DefaultItemAnimator());
         mRecyclerView.hasFixedSize();
 
+        mRecyclerView.addOnItemTouchListener(new RecyclerItemClickListner(getActivity(), this));
+
+        mRecyclerView.setAdapter(mFavouriteWorkflowsAdapter);
+
         showProgressbar(true);
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflow/FavouriteWorkflowsFragment.java]: move this call to Presenter

File: app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflow/FavouriteWorkflowsFragment.java [hunk 2]
```diff
@@ -121,12 +139,25 @@ public void onClick(View view) {
...
 
     @Override
     public void showEmptyWorkflow() {
         tvNoWorkflowError.setVisibility(View.VISIBLE);
     }
+
+    @Override
+    public void onItemClick(View childView, int position) {
+        Intent intent = new Intent(getActivity(), FavouriteWorkflowDetailActivity.class);
+        intent.putExtra("id", mWorkflowList.get(position).getId());
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflow/FavouriteWorkflowsFragment.java]: Make  public static variable of  putExtra string argument in this class.

File: app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflowdetail/FavouriteWorkflowDetailPresenter.java [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    @Override
+    public void detachView() {
+        super.detachView();
+        if (mSubscriptions != null) mSubscriptions.unsubscribe();
+    }
+
+    public void loadWorkflowDetail(String id) {
+        getMvpView().showProgressbar(true);
+
+        mSubscriptions = mDataManager.getFavoriteDetailWorkflow(id)
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflowdetail/FavouriteWorkflowDetailPresenter.java]: you are subscribing more than one time to Subscription, So replace Subscription to CompositeSubscription.
For Example : https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncclientpayloads/SyncClientPayloadsPresenter.java

**Comment by therajanmaurya** [comment]
@sagar15795 make changes and let @ianwdunlop know to merge.

**Comment by sagar15795** [comment]
@ianwdunlop 
I have added all suggestion.
Please merge it.

Thanks

**Pull request merged**